src/components/shared/form_inputs/useInputModel.ts

```diff
@@ -0,0 +1,23 @@
+import { computed, ref, UnwrapRef, watch, WritableComputedRef } from 'vue';
+
+export function useInputModel<T>(
```

Comment:
This seems to be similar to `useFieldModel`, could you elaborate on similarities and differences, maybe add a docblock what this is for?

Reply:
The `fieldModel` is only for watching updates to it's own v-model so it can pass those updates to the child input. The `inputModel` is a 2 way binding between the actual html input and the outer v-model, it bubbles change events and also watches for changes in it's outer model.